pysatNASA/instruments/methods/_cdf.py

```diff
@@ -437,13 +445,14 @@ def to_pysat(self, flatten_twod=True,
         index = None
         for varname, df in cdata.items():
             if varname not in ('Epoch', 'DATE'):
-                if type(df) == pds.Series:
+                if isinstance(df, pds.Series):
```

Comment:
I get that Python has stated they don't like type checks because code should use duck-typing. If it walks like a duck, etc. I also know that the isinstance is recommended over type for when type checks are put in. I don't get it though. This doesn't have to be answered here but if someone does know I'd love to hear it.

Reply:
Pretty sure I implemented this from either a FutureWarning or a linting suggestion.

Reply:
This is the specific error I get when I run flake8: 
```
E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
```

https://www.flake8rules.com/rules/E721.html